### ISSUE TITLE
Question: is it possible to run a SQL model via all-purpose compute cluster?

### ISSUE THREAD

**Issue by csimplestring** [databricks/dbt-databricks#852]
Hello,  is it possible to run a SQL model via all-purpose compute cluster?

The reason why I want to use a cluster to run SQL model is: in the SQL mode, we are using some python UDFs, which the sql warehouse does not support such a feature.

**Comment by benc-db** [comment]
Yes, all-purpose cluster is supported for all dbt-databricks operations.
